docs/src/main/sphinx/release/release-470.md

```diff
@@ -0,0 +1,102 @@
+# Release 470 (5 Feb 2025)
+
+## General
+
+* Prevent failures of the `ARRAY_HISTOGRAM` function resulting from null values. ({issue}`24765`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+* {{breaking}} Raise minimum runtime requirement to Java 11. ({issue}`23639`)
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+## Druid connector
+
+## Elasticsearch connector
+
+## Exasol connector
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+## Hudi connector
+
+## Iceberg connector
+
+* Fix failure when adding a new column with a name containing a dot. ({issue}`24813`)
```

Comment:
Is this correct @ebyhr ?

Reply:
Correct, thanks for rewording the entry.